mem-03-D.xml

```diff
@@ -1664,11 +1663,11 @@ To answer in the negative, say {Qo':excl} "no, I refuse".</column>
       <column name="entry_name">Daqturaq</column>
       <column name="part_of_speech">n:being</column>
       <column name="definition">d'akturak</column>
-      <column name="definition_de">d'akturak [AUTOTRANSLATED]</column>
```

Comment:
Did you mean to remove the tags for languages other than German? This isn't right for some of the other languages.

Reply:
I assumed that as this is a foreign word, created by the authors of the script and presented as a Klingon word in the show that cannot be translated, it would be identical in all languages. Of course, they might have changed that in a different version of the show, but how and why would anyone do that? ... While talking about this, of course we might consider also changing the English gloss, because as Okrand said _"this is the Klingon pronunciation"_. Technically, there is no such thing as a "d'akturak". I suggest using the English word "iceman", which then of course would confirm you saying it's something else in other languages.

Reply:
Okrand was smart avoiding to say this is a Klingon word, because it's not clear from the TV show. Dax said _"I used to call you D'akturak. "Ice man." Because [...] You were the hardest man I ever faced across the negotiating table. "_ - Since she is a Trill, she might have used a Trill word to name him "ice man". (although I think it sounds very Klingon)

Reply:
> I assumed that as this is a foreign word, created by the authors of the script and presented as a Klingon word in the show that cannot be translated, it would be identical in all languages.

One _cannot_ assume this in general. In particular, Persian, Russian, and Chinese do not use the Latin script.

See the entries for "James T. Kirk" and "Jean-Luc Picard" for example.

![Screenshot 2020-07-31 at 11 39 04](https://user-images.githubusercontent.com/11312380/89023317-ac85fe00-d323-11ea-85b9-a1357bae1852.png)

![Screenshot 2020-07-31 at 11 39 18](https://user-images.githubusercontent.com/11312380/89023327-b1e34880-d323-11ea-86f1-14e300cd6901.png)

Reply:
Okay, I admit I didn't think about the different writing system used. As a replacement proverb, I've added the Russian spelling; I don't know where to find the others though.

Reply:
Note that Portuguese has been reviewed (see immediately below).